src/api/treeTrackerApi.js

```diff
@@ -113,9 +122,15 @@ export default {
       .catch(handleError);
   },
   getCaptureById(id) {
+    const lbFilter = {
+      order: [`${'asc'} ${'id'}`],
```

Comment:
`order` doesn't make sense here. This query only returns one record.

Reply:
In fact, this endpoint ignores the param string, so the filter is redundant here. The whole record is always returned for a query by ID.